src/applications/mhv-secure-messaging/tests/components/DeleteDraft.unit.spec.jsx

```diff
@@ -85,23 +85,18 @@ describe('Delete Draft component', () => {
       },
     );
     fireEvent.click(screen.getByTestId('delete-draft-button'));
-    expect(screen.getByTestId('delete-draft-modal')).to.be.visible;
-    expect(
-      screen.findByText(Prompts.Draft.DELETE_DRAFT_CONFIRM, {
-        exact: true,
-      }),
-    );
-    expect(
-      screen.findByText(Prompts.Draft.DELETE_DRAFT_CONFIRM_NOTE, {
-        exact: true,
-      }),
-    );
-    expect(screen.findByText('Delete draft', { exact: true }));
-    expect(screen.findByText('Cancel', { exact: true }));
-    expect(screen.getByTestId('delete-draft-modal')).to.have.attribute(
-      'visible',
-      'true',
-    );
+    // expect(screen.getByTestId('delete-draft-modal')).to.be.visible;
+    await waitFor(() => {

```

Comment:
Note: This fixes this flaky test. One must wait after firing an event. Also note that not all text is rendered as you get elements like va-button with no text to search.